app/demo-geant-integration/HitClassesLinkDef.h

```diff
@@ -7,6 +7,7 @@
 //! \brief Define the classes added to the ROOT dictionary for app/demo-geant-integration
```

Comment:
This line also needs to be fixed for exceeding the length
```suggestion
//! \brief Define classes added to the demo geant ROOT dictionary
```

Reply:
What is the max length for the brief description?

Reply:
The max line length is ~80

Reply:
Ah.  This is the max source line length.  Because of the prefix this also means that the `\brief` is "limited" to 70ish characters.